src/microbe_stage/systems/MicrobeEmissionSystem.cs

```diff
@@ -348,8 +340,16 @@ private void HandleSlimeSecretion(in Entity entity, ref MicrobeControl control,
                 jet.Active = false;
         }
 
-        control.QueuedSlimeSecretionTime -= delta;
-        if (control.QueuedSlimeSecretionTime < 0)
-            control.QueuedSlimeSecretionTime = 0;
+        DecreaseTimeCountdownValue(ref control.QueuedSlimeSecretionTime, delta);
+    }
+
+    private void DecreaseTimeCountdownValue(ref float countdownValue, float delta)
+    {
+        if (countdownValue > 0)
```

Comment:
Is this if actually good for performance? Checking branches is pretty slow in terms of CPU speed, so it might be faster to always just decrement the value and clamp it to 0 if it is negative.

Reply:
Yeah your right, it's quite difficult to determine which one will be faster. I removed the first condition branch